src/test/java/edu/ucla/library/services/metadata/MetadataSetterTest.java

```diff
@@ -0,0 +1,126 @@
+package edu.ucla.library.services.metadata;
+
+import static com.github.stefanbirkner.systemlambda.SystemLambda.catchSystemExit;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.After;
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.junit.contrib.java.lang.system.SystemErrRule;
+
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+
+/*
+ * Unit tests for MetadataSetter.java
+ */
+public class MetadataSetterTest {
+
+  /**
+   * Path to CSV file to be updated.
+  */
+    private static final String CSV_PATH = "src/test/resources/";
+
+  /**
+   * CSV file name.
+  */
+    private static final String CSV_NAME = "sales.csv";
+
+  /**
+   * Path to media files to be read.
+  */
+    private static final String MEDIA_PATH = "src/test/resources/media/";
+
+  /**
+   * Path to ffmpeg probe utility.
+  */
+    private static final String FFMPEG_PATH = "/usr/bin/ffprobe";
+
+  /**
+   * Path where modified CSV file(s) will be written.
+  */
+    private static final String OUTPUT_PATH = "/tmp/";
+
+  /**
+   * Fake path for testing missing parameters.
+  */
+    private static final String FAKE_PATH = "/not/there/";
+
+  /**
+   * Return code for app success.
+  */
+    private static final int SUCCESS = 0;
+
+  /**
+   * Return code for nonexitent file/directory.
+  */
+    private static final int NO_FILE = 101;
+
+  /**
+   * Return code for nonexistent ffprope.
+  */
+    private static final int NO_PROBE = 102;
+
+  /**
+   * rule that allows catching System.err output in tests.
+  */
+    @Rule
+    public final SystemErrRule mySystemErrRule = new SystemErrRule().enableLog();
```

Comment:
Just in case you're using the System.err specifically so you can test those outputs, that's also possible with logging. For instance:

https://github.com/ksclarke/freelib-utils/blob/main/src/test/java/info/freelibrary/util/LoggerTest.java#L63
https://github.com/ksclarke/freelib-utils/blob/main/src/test/java/info/freelibrary/util/test/TestUtils.java#L24

I should create a JUnit Rule for that, just like this other person has done with the System.out/err stuff. That'd make it easily reusable.